backend/src/components/Events/model.ts

```diff
@@ -11,7 +11,8 @@ export interface IEventsnModel extends Document {
     maxScore: number;
     tasks: Types.ObjectId[];
     users: Types.ObjectId[];
-    participations: Types.ObjectId[];
+	participations: Types.ObjectId[];
```

Comment:
Looks like need to sync breaks.
![image](https://user-images.githubusercontent.com/25751683/106248190-b11f6680-6221-11eb-9beb-9f6c06c02cca.png)